todo_app/data/trello_items.py

```diff
@@ -0,0 +1,111 @@
+from todo_app.flask_config import Config
+import requests
+import json
+
+
+class Item:
+    def __init__(self, id, name, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.status = status
+
+    @classmethod
+    def from_trello_card(cls, card):
+        return cls(card['id'], card['name'], card['status'])
+    
+    @classmethod
+    def get_items(cls):
+        cards = TrelloItems().get_cards()
+        return [Item.from_trello_card(card) for card in cards]
```

Comment:
This works but means this class is doing two quite separate things - both representing a single Item, and providing a method that lets you access Trello in order to load the data into the Item format.

Given that the `TrelloItems` service has to handle most of this logic, I'd invert the dependency here and have `TrelloItems` create `Item` objects, rather than having the `Item` class use `TrelloItems` in order to create Items

That way, this `Item` class is a really simple data object just capturing the state of an existing Item, whilst `TrelloItems` can focus on the logic of talking to Trello

E.g. that might look like:
* Have the `get_cards` method convert to Items directly itself:
```python
        cards = []
        for card in response.json():
            item = Item.from_trello_card({
                'id': int(card['idShort']),
                'status': self.get_status_from_list_id(card['idList']),
                'name': card['name'],
                'card_id': card['id'],
                'list_id': card['idList']
            })
            cards.append(item)
        return cards
```
* Remove the `get_items` function on this Item class
* Switch `app.py` over so it calls `trello.get_cards()` instead of `Item.get_items()`